src/components/clusterConfiguration/ClusterValidationSection.tsx

```diff
@@ -104,6 +106,11 @@ const ClusterValidationSection: React.FC<ClusterValidationSectionProps> = ({
               </Flex>
             </Alert>
           )}
+          {!isContentToDisplay && (
```

Comment:
Improves UX in corner cases or bugs, otherwise that should be covered by cluster validations.
Anyway, this is not happening recently, i.e. due to `sufficient-masters-count` workaround.